### PR TITLE
Add build job for issuer front end.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Add a docker build/publish job for the issuer-front-end.

## Checklist

- [ ] My code follows the style of this project.
- [ ] The code compiles without warnings.
- [ ] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: .github/workflows/release-test-issuer-frontend-docker.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    runs-on: ubuntu-latest
+    environment: testnet-builds
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      # Uses the `docker/login-action` action to log in to the Container registry.
+      - name: Log in to the Container registry
+        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
```
[the remark this answers is not in the document]
Comment (abizjak) [reply .github/workflows/release-test-issuer-frontend-docker.yml:27]: It is safer to use it like this.

But it is true we usually don't pay attention to this.

File: .github/workflows/release-test-issuer-frontend-docker.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Extract tag from package.json
+        id: meta
+        run: |
+          export VERSION=$(jq -r .version test-tools/issuer-front-end/package.json)
+          export FULL_IMAGE_TAG="${{ env.REGISTRY }}/concordium/$IMAGE_NAME:$VERSION"
+          echo "::notice FULL_IMAGE_TAG=${FULL_IMAGE_TAG}"
+          # Make sure the image does not exist. Abort if we can retrieve any metadata.
+          if docker manifest inspect docker.io/concordium/issuer-front-end:1.0.14 > /dev/null; then
```
[the remark this answers is not in the document]
Comment (abizjak) [reply .github/workflows/release-test-issuer-frontend-docker.yml:39]: Ah.

File: issuer-front-end/README.md [hunk 3]
```diff
@@ -80,3 +80,11 @@ docker run -it -d -p 8080:80 --name web issuer-front-end:3.0.0
 ```
 
 Open http://127.0.0.1:8080 in your browser.
+
+## Build and push to DockerHub
+
+To build the docker image and push to dockerhub use the
+[test-issuer-front-end](https://github.com/Concordium/concordium-web3id/blob/main/.github/workflows/release-test-issuer-front-end.yaml)
```
[the remark this answers is not in the document]
Comment (abizjak) [reply issuer-front-end/README.md]: Too much renaming :/

File: .github/workflows/release-test-issuer-frontend-docker.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (DOBEN) [reply .github/workflows/release-test-issuer-frontend-docker.yml:27]: makes sense for the release action.

**Pull request merged**